templates/bills/latest_actions.html

```diff
@@ -30,21 +52,62 @@
 </script>
 
 {% endblock %}
+>>>>>>> 815b13c2781b78a3d9a20408a581c6e86ff93414
```

Comment:
when things like this appear it means you have merge conflicts, these need to be resolved before committing the code or it'll cause issues.